src/pages/tool.js

```diff
@@ -0,0 +1,19 @@
+/**
+ *
+ * @author sriram
+ *
+ * A page to access the DE tools.
+ *
+ */
+
+import React from "react";
+
+import Listing from "components/tools/listing/Listing";
+
+export default function Tools() {
```

Comment:
Since the apps and analyses pages are plural, maybe for consistency this file should be renamed to `/pages/tools`? I think the navigation constant is also `tools`.

I checked this out locally and it seems somehow both `/tool` and `/tools` are working.

Reply:
Yup. I created `tool` page by mistake without realizing there was already a `tools` page. `tool` page should be deleted in #248.